### PR TITLE
[add] user table with points filtration

### PR DESCRIPTION
**Pull request by MikeW210**
[add] added JSON Api which gets data about users (id, nick ,points)
[add] added new column (points) to users table 
[add] in user-table.jsx i render Table component (table.jsx) which creates and shows all users on server in one table sorted my amount of points

### CODE REVIEW

File: app/assets/javascripts/users.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (hirokiraj) [inline app/assets/javascripts/users.js]: if this is an unused file let's remove it

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (hirokiraj) [inline app/assets/stylesheets/users.scss]: if this is an unused file let's remove it

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,9 +1,7 @@
...
-        @user = User.find(params[:id])
-    end
-    def index
-        
-    end
-  end
-  
\ No newline at end of file
+
+def index
```
Comment (hirokiraj) [inline app/controllers/users_controller.rb]: if this method is unused we can safely remove it

File: app/helpers/users_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (hirokiraj) [inline app/helpers/users_helper.rb]: if this is an unused file let's remove it

File: spec/controllers/users_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
```
Comment (hirokiraj) [inline spec/controllers/users_controller_spec.rb]: provide user model tests

File: spec/helpers/users_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+# describe UsersHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe UsersHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (hirokiraj) [inline spec/helpers/users_helper_spec.rb]: if this is an unused file let's remove it

File: db/migrate/20190717112841_add_poins_to_user.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+class AddPoinsToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :points, :float
```
Comment (hirokiraj) [inline db/migrate/20190717112841_add_poins_to_user.rb]: maybe we should have a default here

File: app/javascript/components/table.jsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import logo from "../../../serduszeczko.svg";
+// import Heart from './heart'
```
Comment (Vakrim) [inline app/javascript/components/table.jsx]: remove unused code

File: app/javascript/components/table.jsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+                    <th>id</th>
+                    <th>Nick</th>
+                    <th>Points</th>
+                    <th>Fav</th>
+                  </tr> 
+                </thead>
+                <tbody>
+                  {
+                    props.usersData.map(row => (
+                      <tr>
```
Comment (Vakrim) [inline app/javascript/components/table.jsx]: add `key` prop when interating over collection

File: app/javascript/packs/users-table.jsx [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    compare = (a,b) => (b>a) - (b<a);
+    
+    state = {
+        users: []
+    };
+
+    componentWillMount() {
+        axios.get('/api/v1/users.json', {}, { 'Content-Type': 'application/json' })
+            .then(res => {
+                console.log(res.data.data)
```
Comment (Vakrim) [inline app/javascript/packs/users-table.jsx]: remove console logs used for debugging

**Pull request merged**